README.md

```diff
@@ -97,6 +118,43 @@ The plugin also supports the following Teradata connection parameters:
 
 For full description of the connection parameters see https://github.com/Teradata/python-driver#connection-parameters.
 
+### Stored Password Protection
+
+The plugin supports Stored Password Protection feature to specify connection password parameter in encryptred format. The encrypted form of connection password can be specified in your Teradata profile with either `password` connection parameter  or `logdata` connection parameter. 
```

Comment:
Would it not make sense to add one sentence here why anybody would care about stored password protection?

Reply:
Added about Stored Password Protection